### DISCUSSION TITLE
InitialCypher, Node size and Relationship size not working

### DISCUSSION THREAD

**Discussion by loneamarok72** [neo4j-contrib/neovis.js#273] [Q&A]
Hi @thebestnom. I have been stuck with these problems for quite sometime now.

1) "initial_cypher": "match (n:Character)-[r:INTERACTS]->(m:Character) return * limit 25" does not work until I replace the "INTERACTS" with "SEASON1" (The relationship label)

Any help would be greatly appreciated! I have attached the code (Almost the same as the simple-example.html code)

function draw() {
			var config = {
				containerId: "viz1",
				neo4j: {
					serverUrl: "bolt://18.117.169.6:7687",
					serverUser: "neo4j",
					serverPassword: "xxxxxxx"
				},
				labels: {
					"Character": {
						"label": "name",
						"value": "full_pagerank",
						"group": "community"
					}
				},
				relationships: {
					"INTERACTS": {
						"value": "weight"
					}
				},
				"initialCypher": "match (n:Character)-[r:INTERACTS]->(m:Character) return * limit 25"
			};

			viz = new NeoVis.default(config);
			//viz.render();
			cypher = "match (n:Character)-[r:SEASON1]->(m:Character) return * limit 25"
			viz.renderWithCypher(cypher);
			console.log(viz);

	}

**Comment by liudaihai** [comment]
sorry，my English is bad，but i can give you some useful advice
you can do that with me
i use vue,but almost
draw() {
      const config = {
        containerId: "viz",
        neo4j: {
          serverUrl: "bolt://localhost:7687",
          serverUser: "neo4j",
        },
        visConfig: {
          nodes: {
            shape: "ellipse",
            color:"#f0f0f0",
            font: {
              size:30,
              strokeWidth: 5,
              // margin: 0,
              color: "black"
            },
            // size: 80,
            widthConstraint: {
              // maximum: 30,
              minimum: 50
            },
            heightConstraint: {
              maximum: 100,
              minimum: 50
            },
            mass: 2,
            scaling: {
              min: 40,
              max: 60,
              label: {
                enabled: false
              }
            },
            labelHighlightBold: true,
            margin: 2,
            brokenImage: "./WJI2.png",
            shapeProperties:{useImageSize: false}
          },
          edges: {
            arrows: {
              to: {
                enabled: true,
                type: 'arrow',
                scaleFactor: 0.5,
                color:"black"
              }
            },
            arrowStrikethrough: true,
            color: {
              color:"black",

              opacity: 0.5
            },
            // value:1,
            font: {
              size: 30,
              color:"black",
              align: 'middle'
            }
          }
          // nodes: {
          //   shape: 'ellipse',
          //   community:"red"
          // },
          // edges: {
          //   arrows: {
          //     to: {enabled: true}
          //   }
          // },
        },
        labels: {



          "Person":{
            label: "name"

          },
          "Movie":{
            label:"title"
          }

        },

        relationships: {
          Serve_as: {
            label:"property"
            // caption: true,
            // thickness: "score"
          },
          // "WROTE":{
          //   label:"property"
          // },
          // "DIRECTED":{
          //
          // }
        },
        // initialCypher: "match (p:Person)-[r:ACTED_IN]-(o:Movie)-[r1]-(p2:Person) RETURN * LIMIT 10000"
        initialCypher: "MATCH p=()-[r:Serve_as]->() RETURN p LIMIT 700",
        // initialCypher:"MATCH (n)-[r]->(m) RETURN n, r, m;"
      };

      this.neoViz = new NeoVis(config);
      this.neoViz.render();
    },
  },
};

**Reply by liudaihai** [reply]
i hope this helps
